### PR TITLE
[Swift] Fix Parsing issue while executing latest_testflight_build_number action

### PR DESCRIPTION
**Pull request by jhandguy**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

The Ruby `latest_testflight_build_number` lane returns the build number as a String containing double-quotes which lead to failure while parsing to Int. 

See issue https://github.com/fastlane/fastlane/issues/13678

### Description

Replacing all occurrences of double-quotes with an empty String fixed the bug.

Testing could be done by running the `latest_testflight_build_number` lane while using `Fastfile.Swift`.

### CODE REVIEW

**Comment by jhandguy** [comment]
@janpio any way we can get this merged any time soon? 
The `latestTestflightBuildNumber` Swift lane not working is quite a deal breaker for us 😢

**Comment by janpio** [comment]
Please generally don't @ mention people that were not active in a issue or PR before.

That being said, the PR (and underlying issue) is missing information on what exactly is being returned from the ruby method that makes this code necessary.

**Comment by janpio** [comment]
I have no working setup right now, can you show an example of what the `latest_testflight_build_number` action returns?

(FYI: You mean "action" here. The things you write in your `Fastfile` are the lanes.)

**Comment by jhandguy** [comment]
INFO [2018-12-03 14:30:39.94]: --------------------------------------------
INFO [2018-12-03 14:30:39.94]: --- Step: latest_testflight_build_number ---
INFO [2018-12-03 14:30:39.94]: --------------------------------------------
INFO [2018-12-03 14:30:39.94]: Login to App Store Connect (ios-appstore@egym.de)
Loading session from '/Users/jean/.fastlane/spaceship/ios-appstore@egym.de/cookie'
Looking for App Store Connect Team with ID 528841
INFO [2018-12-03 14:30:42.94]: Login successful
INFO [2018-12-03 14:30:43.72]: Fetching the latest build number for version 1.8.9
INFO [2018-12-03 14:30:45.46]: Latest upload for version 1.8.9 is build: 20
DEBUG [2018-12-03 14:30:45.46]: sending {"payload":{"status":"ready_for_next",**"return_object":"\"20\""**,"closure_argument_value":""}}
DEBUG [2018-12-03 14:30:45.46]: received done signal shutting down

The problem comes from the **return_object":"\"20\""**, escaped double-quotes are returned by the Ruby Action.

**Comment by janpio** [comment]
Thanks. I moved further discussion about this back to https://github.com/fastlane/fastlane/issues/13678. Depending on what the result is, this PR might get merged or replaced.

File: fastlane/swift/Fastlane.swift [hunk 1]
```diff
@@ -2065,7 +2065,7 @@ func jira(url: String,
                                                                                                                 RubyCommand.Argument(name: "initial_build_number", value: initialBuildNumber),
                                                                                                                 RubyCommand.Argument(name: "team_id", value: teamId),
                                                                                                                 RubyCommand.Argument(name: "team_name", value: teamName)])
-  return parseInt(fromString: runner.executeCommand(command))
+  return parseInt(fromString: runner.executeCommand(command).replacingOccurrences(of: "\"", with: ""))
```
Comment (taquitos) [inline fastlane/swift/Fastlane.swift]: This file is automatically generated, you'll probably need to take a look at the ruby code in `swift_fastlane_function.rb` or `swift_fastlane_function.rb`

**Review by taquitos — CHANGES_REQUESTED** [review]

**Comment by jhandguy** [comment]
Done! ✅

**Comment by jhandguy** [comment]
Alternatively, the trimming could be done inside the `parseInt(fromString:)` function in the `swift_fastlane_api_generator.rb`

**Comment by jhandguy** [comment]
Or even further, when the `SocketClient` processes the response it already does:
`let responseString = string.trimmingCharacters(in: .whitespacesAndNewlines)`
We could go ahead and trim also punctuation characters:
`let responseString = string.trimmingCharacters(in: .whitespacesAndNewlines).trimmingCharacters(in: .punctuationCharacters)`

**Comment by jhandguy** [comment]
@taquitos did you have time to take a look?
(I'm really sorry to be so pushy... I just believe it should be easy to fix and very low risk 😕)

**Comment by jhandguy** [comment]
@taquitos did you have time to take a look?
(I'm really sorry to be so pushy... I just believe it should be easy to fix and very low risk 😕)

**Comment by jhandguy** [comment]
@taquitos did you have time to take a look?
(I'm really sorry to be so pushy... I just believe it should be easy to fix and very low risk 😕)

**Review by taquitos — COMMENTED** [review]
I wonder if you can add this to the other ones that might have issues, like bool? Just to round this out 😄

**Comment by taquitos** [comment]
No worries! if you could add it to the other return types that might encounter that `""""` I'm happy to approve. Thank you for your fixes!

**Comment by jhandguy** [comment]
@taquitos I just pushed new changes, I moved the trimming to the actual `parseInt` and `parseBool` methods, in case those are reused 👍

**Review by taquitos — APPROVED** [review]
🎈:goat:

**Pull request merged**